cmd/email-provider/server/operations.go

```diff
@@ -16,7 +20,69 @@ func NewServer() *Server {
 }
 
 func (server *Server) SendMessage(ctx context.Context, message *email_provider.SendEmailRequest) (*email_provider.SendEmailResponse, error) {
-	// TODO: send an email to someone here
+	m := mail.NewV3Mail()
+	address := os.Getenv("EMAIL_FROM_ADDRESS")
+	name := os.Getenv("EMAIL_FROM_NAME")
+	e := mail.NewEmail(name, address)
+	m.SetFrom(e)
+
+	p := mail.NewPersonalization()
+	tos := []*mail.Email{
+		mail.NewEmail(os.Getenv("EMAIL_NAME_TO"), os.Getenv("EMAIL_TO_ADDRESS")),
+	}
+	p.AddTos(tos...)
+	p.Subject = os.Getenv("EMAIL_SUBJECT")
+	m.AddPersonalizations(p)
+
+	htmlContent := fmt.Sprintf(
+		`
+		 <strong> First name: </strong> %s <br> 
+		 <strong> Last name: </strong> %s <br>
+		 <strong> Email: </strong> %s <br> 
+		 <strong> Phone: </strong> %s <br><br> 
+		 <strong> Request details: </strong> %s <br> 
+		 <strong> Request service: </strong> %s
+		`,
+		message.Sender.FirstName, message.Sender.LastName, message.Sender.Email, message.Sender.PhoneNumber, message.RequestDetails, message.RequestService)
```

Comment:
could list those variables on multiple lines rather than one line, for readability